bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -1,23 +1,3 @@
 apiVersion: kubecontrolplane.config.openshift.io/v1
-kind: KubeControllerManagerConfig
+kind: EtcdConfig
 extendedArguments:
```

Comment:
this can go too.  I don't think etcd has a config file, does it?

Reply:
The server can have a [config file](https://github.com/etcd-io/etcd/blob/master/etcd.conf.yml.sample)  but I am not sure it would be useful here. I understand.